datavirt/datavirt63-extensions-support-s2i.json

```diff
@@ -745,8 +745,12 @@
                                         "value": "/etc/datavirt-environment/*"
                                     },
                                     {
+                                        "name": "DATAVIRT_TRANSPORT_KEYSTORE_DIR",
+                                        "value": "/etc/datavirt-secret-volume"
+                                    },
```

Comment:
@josefkarasek, let's remove this so it is consistent with using the HTTPS_ as the defaults.  This variable gives users the possibility of using a different keystore for datavirt access (e.g. if they have different keys for internal vs external clients).  From a usability perspective, it's simplest to use the same keystore (and maybe even the same key) as that used for https configuration.